com.github.tsmetana.mpk3-settings.yml

```diff
@@ -0,0 +1,38 @@
+app-id: com.github.tsmetana.mpk3-settings
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+finish-args:
+  - --device=all
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+command: mpk3-settings
+cleanup:
+  - /include
+  - '*.a'
+  - '*.la'
+modules:
+  - name: mpk3-settings
+    buildsystem: autotools
+    post-install:
+      - sed -i'' "s/desktop-id\">com.github.tsmetana.mpk3-settings/desktop-id\">com.github.tsmetana.mpk3-settings.desktop/" "pkg/${FLATPAK_ID}.metainfo.xml"
```

Comment:
please don't use `sed` for this. It has no failure mode and is too obscure.

Reply:
fixed